src/app/app.component.ts

```diff
@@ -25,6 +25,8 @@ import { isPlatformBrowser } from '@angular/common';
 
 import * as $ from 'jquery';
 import * as search_api from 'usgs-search-api';
+import { UserService } from './services/user.service';
```

Comment:
please use the shortcut reference syntax of `@services/user.service` to avoid potential path issues in the future. The `paths` variable of `tsconfig.json` contains all of these shortcuts so if the repo's files are ever restructured, we need only update it on one place.

Reply:
Done